binaries/cli/src/template/rust/mod.rs

```diff
@@ -81,7 +82,9 @@ fn create_operator(name: String, path: Option<PathBuf>) -> Result<(), eyre::ErrR
     fs::create_dir(&src)
         .with_context(|| format!("failed to create directory `{}`", src.display()))?;
 
-    let cargo_toml = CARGO_TOML.replace("___name___", &name);
+    let cargo_toml = CARGO_TOML
+        .replace("___name___", &name)
+        .replace("___version___", &VERSION);
```

Comment:
Good idea!

Reply:
Thanks! We will have to push a new tag on release branch. But that should be doable.

Reply:
Ah, we need that tag already for the CI?

> The tag name will have to be of the same format as cargo version so without "v", directly: "0.1.3"

Couldn't we append the `v` easily?

Reply:
Yes, the idea is that we might push change to the cli, the coordinator, or the runtime, and we should try to check if it break the natural flow of the cli. This include pushing new tag when we have breaking change. 

I'll add the v prefix in the template.